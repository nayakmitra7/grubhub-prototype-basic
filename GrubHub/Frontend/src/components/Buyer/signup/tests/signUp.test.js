import React from 'react';
import Signup from '../signup';
import renderer from 'react-test-renderer';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

it('render correctly Login component', () => {  
  const LoginInputComponent = renderer.create(<Signup />).toJSON();
  expect(LoginInputComponent).toMatchSnapshot();
})


