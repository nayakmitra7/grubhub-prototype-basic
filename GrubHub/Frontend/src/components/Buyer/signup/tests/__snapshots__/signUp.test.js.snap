// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Login component 1`] = `
<div>
  
  <div
    class="backgroundImg"
  >
    <div
      class="container"
    >
      <div
        class="login-form"
      >
        <div
          class="main-div"
        >
          <div
            class="panel"
          >
            <h2
              class="heading"
            >
               Create your account
            </h2>
          </div>
          <div
            class=""
          >
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                First Name
              </div>
              <div
                class="col-md-6"
              >
                Last Name
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-6"
              >
                <input
                  class="form-control"
                  name="firstName"
                  onChange={[Function]}
                  type="text"
                />
              </div>
              <div
                class="col-md-6 "
              >
                <input
                  class="form-control"
                  name="lastName"
                  onChange={[Function]}
                  type="text"
                />
              </div>
            </div>
            <br />
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                   Email
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                 
                <input
                  class="form-control email"
                  name="email"
                  onChange={[Function]}
                  type="text"
                />
              </div>
            </div>
            <br />
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                  Password (8 characters minimum)
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                 
                <input
                  class="form-control email"
                  name="password"
                  onChange={[Function]}
                  type="password"
                />
              </div>
            </div>
            <br />
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                Address
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <textarea
                  class="email"
                  name="address"
                  onChange={[Function]}
                  row="3"
                  type="text"
                />
              </div>
            </div>
          </div>
          <br />
          <button
            class="btn btn-danger"
            onClick={[Function]}
          >
            Sign Up
          </button>
          <br />
          <br />
          <br />
          <br />
          Have an account?
          <a
            class=""
            href="/login"
          >
                 Log In
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
